src/mappings/templateLauncher.ts

```diff
@@ -39,7 +39,7 @@ export function handleTemplateAdded(event: TemplateAdded): void {
  * @param event the `TemplateAdded` event
  */
 export function handleTemplateVerified(event: TemplateVerified): void {
-  let saleTemplate = Schemas.SaleTemplate.load(event.params.templateId.toHexString())
+  let saleTemplate = Schemas.SaleTemplate.load(event.params.templateId.toString())
```

Comment:
I think these were causing the test to fail in the first place! Thank you a lot!